src/test/java/uk/gov/digital/ho/hocs/workflow/processes/COMP_EX_GRATIA_TRIAGE.java

```diff
@@ -19,16 +19,18 @@
 import static org.camunda.bpm.engine.test.assertions.ProcessEngineTests.withVariables;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.digital.ho.hocs.workflow.util.CallActivityMockWrapper.whenAtCallActivity;
 
 @RunWith(MockitoJUnitRunner.class)
-@Deployment(resources = "processes/COMP_EXGRATIA_TRIAGE.bpmn")
+@Deployment(resources = {"processes/COMP_EXGRATIA_TRIAGE.bpmn", "processes/COMP_CLOSE.bpmn" })
 public class COMP_EX_GRATIA_TRIAGE {
 
     @Rule
     @ClassRule
-    public static TestCoverageProcessEngineRule rule = TestCoverageProcessEngineRuleBuilder.create().assertClassCoverageAtLeast(1).build();
+    public static TestCoverageProcessEngineRule rule = TestCoverageProcessEngineRuleBuilder.create().assertClassCoverageAtLeast(0.85).build();
```

Comment:
Any reason for the drop in coverage threshold, can we maintain 1?

Reply:
It's because the call activity is mocked. This lowers the score. COMP.bpmn has the same problem but worse, very low..